src/test/java/teammates/test/cases/action/StudentFeedbackSubmissionEditSaveActionTest.java

```diff
@@ -1381,5 +1381,33 @@ public void testSaveAndUpdateFeedbackParticipantCommentsOnResponse() {
         assertEquals("student1InCourse1@gmail.tmt", frc.commentGiver);
         assertTrue(frc.isCommentFromFeedbackParticipant);
         assertTrue(frc.isVisibilityFollowingFeedbackQuestion);
+
+        ______TS("Test feedback participant comments not allowed in Contribution type questions");
```

Comment:
Are we planning to add comment feature also to contrib question?

Maybe use a text question instead?

Reply:
> Are we planning to add comment feature also to contrib question?

I guess not, because even instructor is not allowed to comment on contrib question.